vnext/ReactWindowsCore/IWebSocketResource.h

```diff
@@ -129,7 +136,7 @@ struct IWebSocketResource {
   /// </param>
   /// <param name="reason">
   /// </param>
-  virtual void Close(CloseCode code, const std::string &reason) = 0;
+  virtual void Close(CloseCode code = CloseCode::None, const std::string &reason = {}) = 0;
```

Comment:
> = {} [](start = 80, length = 5)

This doesn't look right, what does it do? It seems like you are making these optional. Why? And how does &={} work in this case?

Reply:
Just for development convenience.
Close code and reason are merely informative.

Ultimately, they are passed by the owning `WebSocketModule`.